### ISSUE TITLE
Rename this project to russian-earth-vector

### ISSUE THREAD

**Issue by denysdovhan** [nvkelso/natural-earth-vector#813]
I believe **russian-earth-vector** name would better suit this project, since it respects and legitimizes russian claims over sovereign Ukrianian territory. It also endorses illegal occupation and annexation.

No matter how much you say you don’t support russian government, actions speak louder than words.

**Comment by PeterNjeim** [comment]
TL;DR: https://github.com/nvkelso/natural-earth-vector/issues/812#issuecomment-1264449628

Judging by the discussion here: #810, your proposed rename would actually not suit the project, as Natural Earth goes directly against Russia's claims.

If you want to see Russia's perspective, then they do offer a Russia POV file. Luckily that's not the default.

Your abusive language is a violation of GitHub ToS. It's clear you're here to not engage in the facts but instead make unevidenced claims of "support" or "endorsement" when no such actions were made (in fact the opposite is true, condemnation is the only thing that was done).

Creating a map that shows what's on the ground is the only right thing to do. Anything else would be respecting the foreign policy of another country, which is inherently not a neutral point of view, and therefore biased and wrong (wrong not in a moral sense, wrong in a factual sense).

You're committing a guilt by association logical fallacy. It's a common one among social media users. A little logic could've prevented you from making your false statements, but it's too little too late.

I won't be responding to this issue since you make the same arguments as in the issue I linked (which means, you've made no arguments). I'm also not responding to the issue I linked since the discussion ended with a resounding victory for Natural Earth (the opposition could only come up with logical fallacies, nothing of substance).

**Comment by webknjaz** [comment]
> russian-earth-vector

Another good name could be toxic-earth-vector, or maybe ruscist-earth-vector.

**Comment by monfera** [comment]
> which is inherently not a neutral point of view

There's no neutral point of view in case of political maps. Despite the name "Natural Earth", it's not just nature focused, and the presence of any kind of country borders, esp. the default one, makes it a set of political maps. A value-based choice is made by those in control. The current policy happens to align with the Kremlin's interests. So, apparently, the rule of Natural Earth, as set by few maintainers, contradicts the rule that's established at a global scale, by a more representative community, under the UN. Let's agree that rules are political, and serve some purpose, whether it's that of global institutions or that of default country border maintainers in a repo.

**Comment by PeterNjeim** [comment]
@monfera You're repeating the same points that I already debunked in [my essay that you never responded to from several months ago](https://github.com/nvkelso/natural-earth-vector/issues/812#issuecomment-1265871415). For that reason I gave a "dislike" and won't repeat myself.
